external-crates/move/crates/move-compiler/tests/move_2024/naming/syntax_methods_invalid.exp

```diff
@@ -0,0 +1,522 @@
+   ┌─ tests/move_2024/naming/syntax_methods_invalid.move:88:46
+   │
+88 │     public fun lookup_mut<A,B>(s: &mut S<A>, i: B): &mut A { abort 0 }
+   │                                              ^ Unused parameter 'i'. Consider removing or prefixing with an underscore: '_i'
+   │
+   = This warning can be suppressed with '#[allow(unused_variable)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+warning[W09001]: unused alias
+   ┌─ tests/move_2024/naming/syntax_methods_invalid.move:92:14
+   │
+92 │     use std::vector;
+   │              ^^^^^^ Unused 'use' of alias 'vector'. Consider removing it
+   │
+   = This alias does not shadow the built-in type 'vector' in type annotations.
+   = This warning can be suppressed with '#[allow(unused_use)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+warning[W09002]: unused variable
+   ┌─ tests/move_2024/naming/syntax_methods_invalid.move:96:32
+   │
+96 │     public fun lookup_t<T,U,V>(s: &S<T>, i: U, j: V): &u64 { abort 0 }
+   │                                ^ Unused parameter 's'. Consider removing or prefixing with an underscore: '_s'
+   │
+   = This warning can be suppressed with '#[allow(unused_variable)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+warning[W09002]: unused variable
+   ┌─ tests/move_2024/naming/syntax_methods_invalid.move:96:42
+   │
+96 │     public fun lookup_t<T,U,V>(s: &S<T>, i: U, j: V): &u64 { abort 0 }
+   │                                          ^ Unused parameter 'i'. Consider removing or prefixing with an underscore: '_i'
+   │
+   = This warning can be suppressed with '#[allow(unused_variable)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+warning[W09002]: unused variable
+   ┌─ tests/move_2024/naming/syntax_methods_invalid.move:96:48
+   │
+96 │     public fun lookup_t<T,U,V>(s: &S<T>, i: U, j: V): &u64 { abort 0 }
+   │                                                ^ Unused parameter 'j'. Consider removing or prefixing with an underscore: '_j'
+   │
+   = This warning can be suppressed with '#[allow(unused_variable)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+warning[W09002]: unused variable
+   ┌─ tests/move_2024/naming/syntax_methods_invalid.move:99:34
+   │
+99 │     public fun lookup_mut<Q,R,S>(s: &mut S<Q>, i: R, j: S): &mut u64 { abort 0 }
+   │                                  ^ Unused parameter 's'. Consider removing or prefixing with an underscore: '_s'
+   │
+   = This warning can be suppressed with '#[allow(unused_variable)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
+
+error[E03006]: unexpected name in this position
+   ┌─ tests/move_2024/naming/syntax_methods_invalid.move:99:42
+   │
+99 │     public fun lookup_mut<Q,R,S>(s: &mut S<Q>, i: R, j: S): &mut u64 { abort 0 }
+   │                                          ^^^^ Generic type parameters cannot take type arguments
```

Comment:
Is this what we wanted to test?

Reply:
This was meant to make sure some other things balanced out, but I split this file up.